cli.sh

```
@@ -1113,30 +1113,165 @@ copy_claude_configs() {
+			full_command="$full_command $arg"
+		done
+
+		# Prompt user for installation
+		local prompt_msg="Install $name MCP server"
+		[ -n "$description" ] && prompt_msg="$prompt_msg ($description)"
+		[ -n "$category" ] && prompt_msg="$prompt_msg [category: $category]"
+
+		if [ "$YES_TO_ALL" = true ]; then
+			# Auto-accept in non-interactive mode
+			log_info "Auto-installing $name (--yes flag)"
+			if claude mcp add --scope user --transport stdio "$server_name" -- $full_command 2>/dev/null; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)